src/App.js

```diff
@@ -2,77 +2,50 @@ import React, { Component } from 'react';
 import './App.css';
 
 class App extends Component {
-  _isMounted = false;
-
   state = {
-    source: 'https://i.imgur.com/VYUdUNJ.png',
+    source: '',
     error: null,
+    loading: true,
```

Comment:
Added a loading state since componentDidMount() is called after the first render, instead of rendering a blank page the app will use this state to show <p>Loading...</p>